lisa/sut_orchestrator/qemu/platform.py

```diff
@@ -65,21 +68,29 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
         self.qemu_platform_runbook = self.runbook.get_extended_runbook(
             QemuPlatformSchema, type_name=QEMU
         )
-        self.libvirt_conn_str = "qemu:///system"
 
-        if self.qemu_platform_runbook.is_host_remote():
-            self.host_node = RemoteNode(schema.Node(name="qemu-host"), 0, "qemu-host")
+    def _prepare_environment(self, environment: Environment, log: Logger) -> bool:
+        host = self.qemu_platform_runbook.host
+        if host.is_remote():
+            assert host.address
+            assert host.username
+            assert host.private_key_file
+
+            self.host_node = RemoteNode(
+                runbook=schema.Node(name="qemu-host"),
+                index=0,
```

Comment:
You should probably set the `index` to `-1` to avoid confusion with the actual test nodes.

Reply:
Confusion where? In the logs? Isn't the name enough?

Reply:
At least in the other code I've seen, nodes that aren't part of the test environment get an index of `-1`.